haiku-apps/heidi/heidi-1.0.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="Native IDE for Haiku"
+DESCRIPTION="Heidi is a useful IDE for Haiku where coders can edit, \
+build, run, and debug their code projects all in one window."
+HOMEPAGE="https://github.com/waddlesplash/Heidi"
+COPYRIGHT="2014 Austin Cavalier"
```

Comment:
The **Help** :arrow_right: **About...** menu also mentions "Kacper Kacper", so maybe you can (or should?) mention him too.

CC: @waddlesplash, @KapiX.

Reply:
I only added the Scintilla editor plugin, I don't care.